types/react-native/index.d.ts

```diff
@@ -6790,6 +6790,14 @@ export class ScrollView extends ScrollViewBase {
      * @deprecated Use scrollTo instead
      */
     scrollWithoutAnimationTo?: (y: number, x: number) => void;
+    
+    /**
+     * This function sends props straight to native. They will not participate in
+     * future diff process - this means that if you do not include them in the
+     * next render, they will remain active (see [Direct
+     * Manipulation](docs/direct-manipulation.html)).
+     */
+    setNativeProps(nativeProps: Object): void;
```

Comment:
Shouldn’t this be the primitive `object`?
```suggestion
    setNativeProps(nativeProps: object): void;
```

In addition, I think we should change the link to the docs to not be relative but absolute, as the types are coming from a different package than the React Native docs :)

Do you have a link to the source code where this function is defined? Aaand can we add a test for this? :D

Reply:
This was mostly copied from the other `setNativeProps` method in NativeMethods. I'll add your suggestions. Regarding the test, I'll try my best :D